assembl/tasks/sudoer.py

```diff
@@ -0,0 +1,307 @@
+import os
+import sys
+from os.path import exists, join
+
+from invoke import task
+from common import (
+    fill_template,
+    is_integration_env,
+    delete_foreign_tasks,
+    get_venv_site_packages,
+    exists as sudo_exists)
+
+# try:
+#     # invoke 0.11
+#     from invoke import ctask as task
+# except ImportError:
+#     from invoke import task
+
+core_dependencies = [
+    'apt-transport-https',
+    'automake',
+    'bison',
+    'build-essential',
+    'curl',
+    'flex',
+    'gawk',
+    'git',
+    'gperf',
+    'graphviz',
+    'libcurl4-openssl-dev',
+    'libffi-dev',
+    'libgraphviz-dev',
+    'libhiredis-dev',
+    'libmemcached-dev',
+    'libpq-dev',
+    'libreadline-dev',
+    'libssl-dev',
+    'libxml2-dev',
+    'libxmlsec1-dev',
+    'libxslt1-dev',
+    'libzmq3-dev',
+    'pkg-config',
+    'python-dev',
+    'rsync',
+    'wget',
+]
+
+python_dependencies = [
+    'python-dev',
+    'python-pip',
+    'build-essential'
+]
+
+
+@task
+def install_python(c):
```

Comment:
Is this task necessary, I thought I saw you install python manually with apt-get in your scripts

Reply:
No, original intent was to use this to create our AMIs. I think, for centralizing, it should still be as part of our longer term goal. Here is the line that this inv command would replace: 
https://gitlab.com/assembl/terraform/blob/dev/Infrastructure/Packer/BluenoveGolden/scripts/golden_bootstrap.sh#L7